packages/flutter/lib/src/material/material_localizations.dart

```diff
@@ -114,6 +114,11 @@ abstract class MaterialLocalizations {
   /// See also: https://material.io/guidelines/style/typography.html
   TextTheme get localTextGeometry;
 
+  /// Formats a [number] using local decimal number format.
```

Comment:
Since the first line mostly reiterates the method signature, you could just write:  Formats [number] as a decimal, inserting locale-appropriate thousands separators as necessary.

Reply:
Done.